### PR TITLE
Feature/VisualStudio2015 support 

### PR DESCRIPTION
**Pull request by cmarlin**
This branch allow compilation of vmaf project on Visual Studio 2015.
- w32 pthread integration
- classic float declaration
- compiler compatibility stuff

### CODE REVIEW

File: feature/src/adm_tools.c [hunk 1]
```diff
@@ -61,8 +61,17 @@ static double rcp_d(double x)
 static const float dwt2_db2_coeffs_lo_s[4] = { 0.482962913144690, 0.836516303737469, 0.224143868041857, -0.129409522550921 };
 static const float dwt2_db2_coeffs_hi_s[4] = { -0.129409522550921, -0.224143868041857, 0.836516303737469, -0.482962913144690 };
 
+#if 1
+static const double dwt2_db2_coeffs_lo_d[4] = { 0.48296291314469025, 0.83651630373746899, 0.22414386804185735, -0.12940952255092145 };
+#else
 static const double dwt2_db2_coeffs_lo_d[4] = { 0x1.ee8dd4748ca11p-2, 0x1.ac4bdd6e3f184p-1, 0x1.cb0bf0b6b5b13p-3, -0x1.0907dc192d6ddp-3 };
```
Comment (li-zhi) [inline feature/src/adm_tools.c]: What's the point of including these lines? Is there a purpose?

**Comment by li-zhi** [comment]
I've verified the code on Mac, Ubuntu 16, Fedora and OpenSUSE, but not on Windows. @cmarlin can I put you as the point of contact for any Windows-related issues?

File: feature/src/adm_tools.c (hunk 1 above)
Comment (cmarlin) [inline feature/src/adm_tools.c]: VS2015 doesn't support C++17 (http://en.cppreference.com/w/cpp/language/floating_literal) it seems to be a gcc extension. the #if is only to discuss about mantis rounding. Do we need exact mantis value ?

**Comment by cmarlin** [comment]
Sure, I can fix windows builds if broken, and other stuffs.

----- Mail original -----
De: "Zhi Li" <notifications@github.com>
À: "Netflix/vmaf" <vmaf@noreply.github.com>
Cc: "cmarlin" <cmarlin@free.fr>, "Mention" <mention@noreply.github.com>
Envoyé: Jeudi 13 Juillet 2017 06:22:10
Objet: Re: [Netflix/vmaf] Feature/VisualStudio2015 support (#92)



I've verified the code on Mac, Ubuntu 16, Fedora and OpenSUSE, but not on Windows. @cmarlin can I put you as the point of contact for any Windows-related issues?

**Comment by li-zhi** [comment]
@cmarlin I would like to reach out to you regarding an issue of VMAF re-licensing. Would you please email me at zli@netflix.com with your personal email address? Thanks!

**Pull request merged**